src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -111,7 +111,7 @@ public static void fillWorkspaceWithTestData(Workspace workspace) {
 		VariableElement sampleVariable1 = new VariableElement();
 		sampleVariable1.setName("test");
 		sampleVariable1.setValue("true");
-		sampleVariable1.setType(VariableElementType.LOGIC);
+		sampleVariable1.setType(VariableElementType.LOGIC.getType());

```

Comment:
It is strange that VariableElementType has method getType, I think this should probably be getName or getTypeName or something like this, getType on type is a bit too generic